homeassistant/components/switch/rpi_rf.py

```diff
@@ -21,13 +21,16 @@
 CONF_GPIO = 'gpio'
 CONF_PROTOCOL = 'protocol'
 CONF_PULSELENGTH = 'pulselength'
+CONF_REPEATS = 'repeats'
```

Comment:
In other components we use signal_repetitions, so might good to keep the consistency. 
https://github.com/home-assistant/home-assistant/blob/08591aacc9b16747f9b3232e523076f8c8ae01a5/homeassistant/components/rfxtrx.py#L31